### DISCUSSION TITLE
Timeout in different countries?

### DISCUSSION THREAD

**Discussion by Therealjosephchrzempiec** [networked-aframe/networked-aframe#398] [Q&A]
Hello, I'm trying to get my friend to login to my networked-aframe and he is timing out. By another friend in the same USA as me can get into it fine. I know this is not a networked-aframe problem. I was wondering If anyone ran into this problem before and was able to fix this problem?

Joseph

**Comment by vincentfretin** [comment]
This person may need to go through a turn server. See https://github.com/networked-aframe/networked-aframe/issues/363

**Reply by Therealjosephchrzempiec** [reply]
I'm looking into that Thank you.
